src/extensions/RichText.js

```diff
@@ -43,6 +43,7 @@ import TaskList from './../nodes/TaskList.js'
 import Text from '@tiptap/extension-text'
 import TrailingNode from './../nodes/TrailingNode.js'
 /* eslint-enable import/no-named-as-default */
+import TextDirection from 'tiptap-text-direction'
```

Comment:
Ah... this also triggers the `import/no-named-as-default` eslint rule. So please move it one line up so it's in the `eslint-disable` block above.

Thanks!